core/src/test/scala/org/apache/spark/SparkThrowableSuite.scala

```diff
@@ -45,7 +45,6 @@ class SparkThrowableSuite extends SparkFunSuite {
         "core/testOnly *SparkThrowableSuite -- -t \"Error classes are correctly formatted\""
    }}}
    */
-  private val regenerateGoldenFiles: Boolean = System.getenv("SPARK_GENERATE_GOLDEN_FILES") == "1"
```

Comment:
sorry I may miss some context. What problem do we fix by moving this to `SparkFunSuite`?

Reply:
`TPCDSBase` needs this parameter:
<img width="1353" alt="image" src="https://user-images.githubusercontent.com/5399861/185517872-92adabf2-bc12-488b-906f-b382e5c01d71.png">